compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/codegen/ExpressionCodegen.kt

```diff
@@ -778,9 +777,13 @@ class ExpressionCodegen(
         jump.markLineNumber(startOffset = true)
         val endLabel = Label()
         val stackElement = unwindBlockStack(endLabel, data) { it is LoopInfo && it.loop == jump.loop } as LoopInfo?
-            ?: throw AssertionError("Target label for break/continue not found")
-        mv.fixStackAndJump(if (jump is IrBreak) stackElement.breakLabel else stackElement.continueLabel)
-        mv.mark(endLabel)
+        if (stackElement == null) {
```

Comment:
Is it possible to search proper loop outside (in inlinedInto codegens)?. And if not present throw exception, otherwise generate global return flag?

Reply:
This data is stored not in the codegen itself, but in `BlockInfo`, which we don't have here. Though I suppose ExpressionCodegen could easily be changed to store the block stack in a field instead of passing it around as a parameter.

Reply:
Or pass it to ExpressionCodegen constructor

Reply:
Actually this is minor, because there should be some IR validation and this structure should be consistent. This request could be simply skipped